.github/workflows/build.yml

```diff
@@ -0,0 +1,48 @@
+name: build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    # Service containers to run with `container-job`
+    services:
+      # Label used to access the service container
+      postgres:
+        # Docker Hub image
+        image: postgres
+        # Provide the password for postgres
+        env:
+          POSTGRES_PASSWORD: placeholder_db_password
```

Comment:
What's this for / do I need to update this for HATS?

Reply:
This is what the default is in the example properties file, so I set it to match. Shouldn't be any changes required for HATS specifically.